README.md

```diff
@@ -15,39 +18,44 @@ NVIDIA Merlin is a scalable and GPU-accelerated solution, making it easy to buil
-- process datasets that don’t fit within the GPU or CPU memory by streaming from the disk.
-- prepare batches asynchronously into the GPU to avoid CPU-GPU communication.
-- integrate easily into existing TensorFlow or PyTorch training pipelines by using a similar API.
-
 **[HugeCTR](https://github.com/NVIDIA-Merlin/HugeCTR)**<br>
-HugeCTR is a GPU-accelerated framework designed to estimate click-through rates and distribute training across multiple GPUs and nodes. HugeCTR contains optimized dataloaders that can be used to prepare batches with GPU-acceleration. In addition, HugeCTR is capable of scaling large deep learning recommendation models. The neural network architectures often contain large embedding tables that represent hundreds of millions of users and items. These embedding tables can easily exceed the CPU and GPU memory. HugeCTR provides strategies for scaling large embedding tables beyond available memory. With HugeCTR, you can:
+HugeCTR is a GPU-accelerated training framework designed to scale large deep learning recommendation models by distributing training across multiple GPUs and nodes. HugeCTR contains optimized dataloaders with GPU-acceleration and provides strategies for scaling large embedding tables beyond available memory. With HugeCTR, you can:
 - scale embedding tables over multiple GPUs or nodes.
 - load a subset of an embedding table into the GPU in a coarse grained, on-demand manner during the training stage.
 
-**[Triton](https://github.com/triton-inference-server/server)**<br>
-NVTabular and HugeCTR both support the Triton Inference Server to provide GPU-accelerated inference. The Triton Inference Server is open-source inference serving software that can be used to simplify the deployment of trained AI models from any framework to production. With Triton, you can:
-- deploy NVTabular ETL workflows and trained deep learning models to production with a few lines of code.
-- deploy an ensemble of NVTabular ETL and trained deep learning models to ensure that the same data transformations are applied in production.
-- deploy models concurrently on GPUs to maximize utilization.
-- enable low latency inferencing in real time or batch inferencing to maximize GPU and CPU utilization. 
-- scale the production environment with Kubernetes for orchestration, metrics, and auto-scaling using a Docker container.
+**[Merlin Models](https://github.com/NVIDIA-Merlin/models)** [![PyPI version shields.io](https://img.shields.io/pypi/v/merlin-models.svg)](https://pypi.python.org/pypi/merlin-models/)<br>
+The Merlin Models library provides standard models for recommender systems with an aim for high quality implementations that range from classic machine learning models to highly-advanced deep learning models. It features:
+ - Performant dataloaders for Tensorflow, PyTorch and HugeCTR, accelerating ranking model training by up to 10x.
```

Comment:
The preceding lists began with verbs--kind of answering what the library _does_.  This bullet list covers what the library _is_.

Plan "A":
- initial lowercase each bullet, such as "performant..."

Plan "B": (requires playing true or false):

s/It features:/With Merlin Models, you can:/

- accelerates raking model training by up to 10x by providing performant data loaders for TensorFlow, PyTorch, and HugeCTR.
- facilitates rapid iteration of feature engineering...
- provides connectable building blocks...

(OK, maybe only the first one was intimidating.)